sdk/appconfiguration/app-configuration/test/public/auth.spec.ts

```diff
@@ -32,6 +32,19 @@ describe("Authentication", () => {
   });
 
   it("token authentication works", async function() {
+    switch (process.env["AZURE_AUTHORITY_HOST"]) {
+      case "https://login.microsoftonline.us":
+        credsAndEndpoint.endpoint = credsAndEndpoint.endpoint
+          .toString()
+          .replace(".azconfig.io", ".azconfig.azure.us");
+        break;
+      case "https://login.chinacloudapi.cn":
+        credsAndEndpoint.endpoint = credsAndEndpoint.endpoint
+          .toString()
+          .replace(".azconfig.io", ".azconfig.azure.cn");
+        break;
+    }
+
```

Comment:
@benbp I would need this snippet.
Somehow,     "azConfigEndpointSuffix" seems to remain as ".azconfig.io".

Reply:
Removing the snippet and testing.
https://dev.azure.com/azure-sdk/internal/_build/results?buildId=1098508&view=results